taskipy/task_runner.py

```diff
@@ -93,14 +97,36 @@ def send_signal_to_task_process(signum: int, _frame) -> None:
 
         return process.returncode
 
-    def __pre_task(self, task_name: str):
-        return self.__find_hooks('pre', task_name)
+    def __resolve_variables(self, command: str) -> str:
+        types = self.__get_variable_types(self.__project.variables)
+        resolved_vars = types["nonrecursive"]
+        recursive_vars = types["recursive"]
 
-    def __post_task(self, task_name: str):
-        return self.__find_hooks('post', task_name)
+        while len(recursive_vars) > 0:
+            resolved_count = len(resolved_vars)
```

Comment:
```suggestion
            count_of_previously_resolved_vars = len(resolved_vars)
```

IMO this helps devs understand that we are trying to count how many vars we already resolved before running this iteration. This also makes line 115 more readable:

```python
if len(resolved_vars) == count_of_previously_resolved_vars:
```

wdyt?

Reply:
I agree